tests/foreman/ui/test_activationkey.py

```diff
@@ -426,48 +426,6 @@ def test_positive_delete_activation_key_with_system(self):
                 self.activationkey.delete(name)
                 self.assertIsNone(self.activationkey.search_key(name))
 
-    @run_only_on('sat')
```

Comment:
That test duplicates previous ones and moreover uses `self.content_views.delete_version` with completely wrong parameters